pandas/_libs/groupby_helper.pxi.in

```diff
@@ -88,7 +89,7 @@ def group_add_{{name}}(ndarray[{{dest_type2}}, ndim=2] out,
 
         for i in range(ncounts):
             for j in range(K):
-                if nobs[i, j] == 0:
+                if nobs[i, j] < min_count:
```

Comment:
does this work for ``min_count==0``?

Reply:
Yeah, `sumx`  starts out as `zeros`, so we just have to avoid setting it to NaN. Same for `prod`, but with ones.